### PR TITLE
🌐 Add Portuguese translation for `docs/pt/docs/advanced/openapi-webhooks.md`

### PR DESCRIPTION
**Pull request by ceb10n**

### CODE REVIEW

File: docs/pt/docs/advanced/openapi-webhooks.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+Note que utilizando webhooks você não está de fato declarando uma **rota** (como `/items/`), o texto que informa é apenas um **identificador** do webhook (o nome do evento), por exemplo em `@app.webhooks.post("new-subscription")`, o nome do webhook é `new-subscription`.
+
+Isto porque espera-se que os **seus usuários** definam o verdadeiro **caminho da URL** onde eles desejam receber a requisição do webhook de algum outra maneira. (e.g. um painel).
+
+### Confira a documentação
+
+Agora você pode iniciar a sua aplicação e ir até <a href="http://127.0.0.1:8000/docs" class="external-link" target="_blank">http://127.0.0.1:8000/docs</a>.
+
+Você verá que a sua documentação possui as *operações de rota* normals e agora também possui alguns **webhooks**:
```
Comment (Joao-Pedro-P-Holanda) [inline docs/pt/docs/advanced/openapi-webhooks.md:49]: typo in "normals"

**Review by Joao-Pedro-P-Holanda — APPROVED** [review]
Great job 🚀, just a little typo in line 49.

**Comment by tiangolo** [comment]
📝 Docs preview for commit 6917d75deb3e8f69db1d015d55f5daed197e5930 at: https://843ccce4.fastapitiangolo.pages.dev

**Comment by alejsdev** [comment]
Nice, thank you! @ceb10n :sunglasses: 

And thanks for the review @Joao-Pedro-P-Holanda :rocket:

**Pull request merged**